.pre-commit-config.yaml

```diff
@@ -11,3 +11,5 @@ repos:
       - id: check-docstring-first
       - id: check-executables-have-shebangs
       - id: check-merge-conflict
+      - id: check-shebang-scripts-are-executable
```

Comment:
I argue against `check-shebang-scripts-are-executable` or at least exclude `*.j2` files. They are mostly not valid scripts before being templated.

Reply:
I've added j2 exclusion.